src/VisualStudio/Core/Def/Options/PackageSettingsPersister.cs

```diff
@@ -42,12 +43,19 @@ private async Task InitializeAsync(CancellationToken cancellationToken)
 
             _lazyRoslynPackage = await RoslynPackage.GetOrLoadAsync(_threadingContext, _serviceProvider, cancellationToken).ConfigureAwait(true);
             Assumes.Present(_lazyRoslynPackage);
-
-            _optionService.RefreshOption(new OptionKey2(SolutionCrawlerOptionsStorage.SolutionBackgroundAnalysisScopeOption), _lazyRoslynPackage.AnalysisScope);
             _lazyRoslynPackage.AnalysisScopeChanged += OnAnalysisScopeChanged;
+
+            await TaskScheduler.Default;
+
+            // make sure we do not access the options or the containing assembly on UI thread:
+            RefreshSolutionBackgroundAnalysisScopeOption();
```

Comment:
Why not? We trigger a bunch of stuff from this?

Reply:
The issue shows assembly load - I assume that's the language server protocol assembly.